diff -urNp -x '*.orig' openldev.org/openldev.desktop openldev/openldev.desktop
--- openldev.org/openldev.desktop	2007-03-28 17:27:59.000000000 +0200
+++ openldev/openldev.desktop	2025-08-08 11:33:33.519584170 +0200
@@ -6,6 +6,6 @@ Name=OpenLDev
 GenericName=OpenLDev
 Comment=Integrated Development Environment (IDE)
 Exec=openldev
-Icon=/usr/share/pixmaps/openldev/openldev48.png
+Icon=openldev
 Terminal=false
-Categories=Application;Development;
+Categories=GTK;GNOME;Development;IDE;
